unit-tests/live/d400/test-auto-limits.py

```diff
@@ -0,0 +1,82 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+# test:device D435I
+# test:device D455
+
+import pyrealsense2 as rs
+from rspy import test
+
+#############################################################################################
+with test.closure("Gain/Exposure auto limits"):
```

Comment:
Split into separate tests - gain toggle one device, gain toggle 2 devices, ae toggle one device, ae toggle 2 devices.
Also are these all the use cases? Test trying to set value when toggle is off etc...